mapfishapp/src/main/java/org/georchestra/mapfishapp/ws/DocController.java

```diff
@@ -41,36 +44,65 @@
  * <br />
  * The implementation should be modified when Spring 3.0 is released. It provides RESTful tools such as URI templates. <br />
  * It could also make easier to create a DocService Factory that create Doc Services with the service name found in the URL. <br />
- * 
+ *
  * @author yoann.buch@gmail.com
  *
  */
 
 // TODO: Implements REST Style controllers when Spring framework 3.0 is released
-  
+
 @Controller
 public class DocController {
 
 	/** the temporary directory used by the document services*/
     private String docTempDir;
 
-    public String getDocTempDir() {return docTempDir;}
-	public void setDocTempDir(String docTempDir) {	this.docTempDir = docTempDir; }
+    @Autowired
+    public GeorchestraConfiguration georchestraConfiguration;
 
-	/** the postgresql connection pool used by the document services*/
+    /** the connection pool used by the document services*/
+    @Autowired
     private ConnectionPool connectionPool;
 
-    public ConnectionPool getConnectionPool() {return connectionPool;}
-	public void setConnectionPool(ConnectionPool connectionPool) {	this.connectionPool = connectionPool; }
+    public void init() throws IOException {
+        if (georchestraConfiguration.activated()) {
+            docTempDir = georchestraConfiguration.getProperty("docTempDir");
```

Comment:
hmm, maybe not, since it seems to be still used for uploads cache (unless we can store them elsewhere)